indigo_app/templates/account/email.html

```diff
@@ -13,8 +13,11 @@ <h5 class="card-header">Email addresses</h5>
   <div class="card-body">
     {% with user.emailaddress_set.all as emails %}
       {% if emails %}
-        <p>You can use any of these email addresses to log in.</p>
-        <p><strong>Note:</strong> We only send email notifications to your <strong>primary</strong> email address.</p>
+        <p>{% trans 'You can use any of these email addresses to log in.' %}</p>
+        <p>
+          <strong>{% trans 'Note:' %}</strong>
+          {% blocktrans %}We only send email notifications to your <strong>primary</strong> email address.{% endblocktrans %}
```

Comment:
could the 'Note: ' not also go into the blocktrans? or do we want 'Note: ' separately to reuse?